bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java

```diff
@@ -431,6 +433,47 @@ public ServerConfiguration setVerifyMetadataOnGc(boolean verifyMetadataOnGC) {
         return this;
     }
 
+    /**
+     * Get whether the bookie is configured to use persistent
+     * entrylogMetadataMap.
+     * @return use persistent entry-log metadata map
+     */
+    public boolean isGcPersistentEntrylogMetadataMapEnabled() {
+        return this.getBoolean(GC_PERSISTENT_ENTRYLOGMETADATA_MAP_ENABLED, true);
```

Comment:
i'm not sure if it is correct to set it to true by default. This will have perf impact, so I dont think making it default behavior is not ideal for new users/ who are unaware of usage characteristics.

Reply:
sure. I will change it to false.